### PR TITLE
Feature/Set up environnement

### PR DESCRIPTION
**Pull request by LeVHoangduc**

### CODE REVIEW

File: blog/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[*]
+indent_size = 2
+indent_style = space
```
Comment (tungtran-agilityio) [inline blog/.editorconfig:3]: add space

File: blog/.eslintignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```
Comment (tungtran-agilityio) [inline blog/.eslintignore:2]: space

File: blog/.editorconfig (hunk 1 above)
Comment (tungtran-agilityio) [reply blog/.editorconfig:3]: Why is your prettier not working?

File: blog/src/App.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#root {
```
Comment (tungtran-agilityio) [inline blog/src/App.css:1]: where are other folder components, utils...?

File: blog/.eslintrc.cjs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      node: {
+        paths: ['src'],
+        extensions: ['.js', '.jsx', '.ts', ',tsx']
+      }
+    }
+  },
+  rules: {
+    'react-refresh/only-export-components': ['warn', { allowConstantExport: true }],
+    'prettier/prettier': [
+      'warn',
```
Comment (tungtran-agilityio) [inline blog/.eslintrc.cjs:32]: you should change it to error

File: blog/package.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (tungtran-agilityio) [inline blog/package.json:1]: You should check dependencies and devDependencies

File: blog/.editorconfig (hunk 1 above)
Comment (LeVHoangduc) [reply blog/.editorconfig:3]: @tungtran-agilityio It means, i need to set up it in my prettier?

File: blog/.eslintignore (hunk 2 above)
Comment (LeVHoangduc) [reply blog/.eslintignore:3]: i have updated it. Please help me to check again @tungtran-agilityio

File: blog/.eslintrc.cjs (hunk 4 above)
Comment (LeVHoangduc) [reply blog/.eslintrc.cjs]: @tungtran-agilityio I have updated it. Please help me to check again

File: blog/package.json (hunk 5 above)
Comment (LeVHoangduc) [reply blog/package.json:1]: @tungtran-agilityio I have checked again and I think it's fine

**Review by tungtran-agilityio — APPROVED** [review]

**Pull request merged**